### PR TITLE
fixups: QoL updates

### PR DESCRIPTION
**Pull request by twilfredo**
This patch series adds:

-  `README` to help new users
-  Updates the `usage` message to print missing options.
- Updates deprecated glibc api call

### CODE REVIEW

**Comment by twilfredo** [comment]
ping!

File: src/gmc-display-config.c [hunk 1]
```diff
@@ -617,7 +617,7 @@ cc_display_mode_get_supported_scales (CcDisplayMode *mode,
                                       int           *n_supported_scales)
 {
   *n_supported_scales = mode->n_supported_scales;
-  return g_memdup (mode->supported_scales,
+  return g_memdup2 (mode->supported_scales,
                    sizeof (double) * mode->n_supported_scales);
```
Comment (jadahl) [inline src/gmc-display-config.c:621]: needs adapting indentation

**Review by jadahl — COMMENTED** [review]
lgtm, just a tiny little line that needs an extra whitespace

File: src/gmc-display-config.c (hunk 1 above)
Comment (twilfredo) [reply src/gmc-display-config.c]: ah woops, fixed :)

**Comment by jadahl** [comment]
thanks!

**Pull request merged**